inst/paper/paper.md

```diff
@@ -50,17 +50,17 @@ link-citations: yes
 
 # Statement of need
 
 The onset of the COVID-19 pandemic in late 2019 has placed pressure on public health and research communities to generate evidence that can help advise national and international policy in order to reduce transmission and mitigate harm. At the same time, there has been a renewed policy and public health emphasis on localised, subnational decision making and implementation [@Hale2021; @Liu2021]. This requires reliable sources of data disaggregated to a fine spatial scale, ideally with few and/or known sources of bias.
 
 At a national level, epidemiological COVID-19 data is available to download from official sources such as the [World Health Organisation (WHO)](https://covid19.who.int/) [@WorldHealthOrganisation] or the [European Centre for Disease Control (ECDC)](https://www.ecdc.europa.eu/en/publications-data/download-todays-data-geographic-distribution-covid-19-cases-worldwide%7D) [@EuropeanCentreforDiseasePreventionandControl]. Many government bodies provide a wider range of country specific data, such as [Public Health England in the United Kingdom](https://coronavirus.data.gov.uk/details/about-data) [@PublicHealthEngland], and this is often the only way to access data at a subnational scale, for example by state, district, or province.
 
-Sometimes collated from a range of national and subnational sources, these data come in a variety of formats, requiring users to check and standardise data before it can be combined or processed for analysis.  This is a particularly time-consuming process for subnational data sets, which are often only available in the originating countries’ languages and require customised methods for downloading and processing. This generates potential for errors through programming mistakes, changes to a dependency package, or unexpected changes to a data source. This can lead to misrepresenting the data in ways which are difficult to identify. At best, an independent data processing workflow only slows down the pace of research and analysis, while at worst it can lead to misleading and erroneous results.
+Sometimes collated from a range of national and subnational sources, these data come in a variety of formats, requiring users to check and standardise data before it can be combined or processed for analysis.  This is a particularly time-consuming process for subnational data sets, which are often only available in the originating countries’ languages and require customised methods for downloading and processing. This generates potential for errors through programming mistakes, changes to functions defined in other packages, or unexpected changes to a data source. This can lead to misrepresenting the data in ways which are difficult to identify. At best, an independent data processing workflow only slows down the pace of research and analysis, while at worst it can lead to misleading and erroneous results.
```

Comment:
I think the previous wording was clearer - lets stick with it and explain to reviewers.